config/crd/bases/scheduling.volcano.sh_podgroups.yaml

```diff
@@ -48,11 +48,7 @@ spec:
                 type: integer
               minResources:
                 additionalProperties:
-                  anyOf:
-                  - type: integer
-                  - type: string
-                  pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                  x-kubernetes-int-or-string: true
+                  type: string
```

Comment:
what's the reason to change the pattern?

Reply:
controller-gen version is different between github  and my env， I update my env to v0.4.1